site/src/components/Link.tsx

```diff
@@ -8,11 +8,13 @@ import NextLink, { LinkProps as NextLinkProps } from "next/link";
 import MuiLink, { LinkProps as MuiLinkProps } from "@mui/material/Link";
 import { styled } from "@mui/material/styles";
 import { Button } from "./Button";
+import { FRONTEND_URL } from "../lib/config";
 
 export const isHrefExternal = (href: string | UrlObject) =>
   typeof href === "string" &&
   (href === "/discord" ||
-    !/^(mailto:|#|\/|https:\/\/blockprotocol\.org)/.test(href));
+    !/^(mailto:|#|\/|https:\/\/blockprotocol\.org)/.test(href)) &&
+  !href.includes(FRONTEND_URL);
```

Comment:
updated this because it treated absolute local urls as external links. An example of where we have to use an absolute local url is `schema.$id` property in entityTypes

Reply:
```suggestion
  !href.startsWith(FRONTEND_URL);
```